packages/desktop/electron/main.js

```diff
@@ -38,10 +56,25 @@ let lastError = {}
  */
 const handleError = (errorType, error, isRenderProcessError) => {
     if (app.isPackaged) {
+        const diagnostics = getDiagnostics()
+
         lastError = {
-            diagnostics: getDiagnostics(),
-            errorType,
+            diagnostics,
             error,
+            errorType,
+        }
+
+        if (SEND_CRASH_REPORTS) {
+            Sentry.captureException(
```

Comment:
Are we not able to use the `captureException` from `../sentry`?

Reply:
We can :+1: